docs/src/main/sphinx/functions/math.rst

```diff
@@ -278,3 +278,15 @@ Cumulative distribution functions
     Compute the Normal cdf with given mean and standard deviation (sd):  P(N < v; mean, sd).
     The mean and value v must be real values and the standard deviation must be a real
     and positive value.
+
+.. function:: chiSquaredCdf(df, value) -> double
+
+    Compute the Chi-square cdf with given df (degrees of freedom) parameter: P(N < value; df).
```

Comment:
Can we expand `cdf` to cumulative distribution function ?

Reply:
Done!